### PR TITLE
Trigger email on user approved

### LINKED ISSUE #163: Set up Email trigger when user is approved (for users who logged in with email)

### PR DESCRIPTION
**Pull request by dalmia**
Fixes #163 

## Summary
- for users who have logged in through their email, when the user status is changed to approved, sends an email trigger notifying them of the approval.

## Test Plan
Tested locally.

## TODOs
- [ ] set up plio gmail account + logo
- [ ] change email creds to plio
- [x] add env variables on staging
- [x] test on staging
- [ ] add env variables on production
- [ ] test on production

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 set -e
 
+# collect static files
+python manage.py collectstatic --no-input
+
```
Comment (deepansh96) [inline entrypoint.sh:6]: this is because we have some HTML/CSS in the code now?

File: .env.example [hunk 2]
```diff
@@ -45,3 +45,16 @@ AUTH0_TOKEN_URL=
...
+EMAIL_HOST='smtp.gmail.com'
+EMAIL_USE_TLS=True
+EMAIL_PORT=587
+DEFAULT_FROM_EMAIL=Plio <no-reply@plio.in>
+EMAIL_HOST_USER= # sender's email-id
+# password associated with above email-id
+# for accounts with 2-factor authentication, you have to create a application-specific
+# passwords. More on that here:
+# https://dev.to/abderrahmanemustapha/how-to-send-email-with-django-and-gmail-in-production-the-right-way-24ab
+EMAIL_HOST_PASSWORD=
```
Comment (deepansh96) [inline .env.example:60]: references

Comment (deepansh96) [reply .env.example:60]: do we want to put the backend waali cheez in env file?
For example, for authentication we had these 
```py
AUTHENTICATION_BACKENDS = (
    "social_core.backends.google.GoogleOAuth2",
    "rest_framework_social_oauth2.backends.DjangoOAuth2",
    "django.c
```

File: users/views.py [hunk 3]
```diff
@@ -1,5 +1,20 @@
...
+import random
+from oauth2_provider.models import AccessToken, Application, RefreshToken
+
+from asgiref.sync import async_to_sync
+from django.contrib.auth import login
+from django.dispatch import receiver
+from django.db.models.signals import post_save, pre_save, post_delete
+from django.core.mail import send_mail
+from django.template.loader import render_to_string
+from channels.layers import get_channel_layer
```
Comment (deepansh96) [inline users/views.py:17]: are these lines just shifted up?

Comment (dalmia) [reply users/views.py:17]: yep

File: users/views.py [hunk 4]
```diff
@@ -197,6 +200,21 @@ def update_user(sender, instance: User, **kwargs):
...
+        # send an email if the user has been approved
+        if instance.email and instance.status == "approved":
+            subject = "Congrats - You're off the Plio waitlist! 🎉"
+            recipient_list = [
+                instance.email,
+            ]
+            html_message = render_to_string("waitlist-approve-email.html")
+            send_mail(
+                subject,
+                None,
```
Comment (deepansh96) [inline users/views.py:212]: = waala structure please

File: entrypoint.sh (hunk 1 above)
Comment (dalmia) [reply entrypoint.sh:6]: I needed this when I was trying to add images. Although I discarded using images in the repo, this line should always be there. It was missed by Vaibhav during the setup.

**Pull request merged**